src/System.Private.Xml/src/System/Xml/Dom/XPathNodeList.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
After processing the file is now pre-pended with EF BB BF. i.e. the BOM for UTF-8 which was not there before.